support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -58,4 +59,26 @@ export const tests: Tests = {
     targetPage: allLandingPagesAndThankyouPages,
     seed: 18,
   },
+
+  localCurrencyTest: {
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'variant',
+      },
+    ],
+    audiences: {
+      DE: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    canRun: () => ['SE', 'CH', 'NO', 'DK'].includes(countryCode),
```

Comment:
Can we do this all just with the audiences object? i.e 

```js
   audiences: {
      SE: {
        offset: 0,
        size: 1,
      },
      CH: {
        offset: 0,
        size: 1,
      },
     // ...
    },
```

Reply:
Unfortunately not @TomPretty, the country value here always defaults to 'DE' for European countries.

Reply:
Although now I'm wondering if this is because certain cookies aren't set when running locally, this might work in production. I'll check now!

Reply:
😨

Reply:
looks like we get the country here: https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/internationalisation/country.js#L587 but I'm not sure why that would fallback to `DE` for European countries

Reply:
@TomPretty I think the `GU_country` and `GU_geo_country` cookies aren't set when running locally, so European countries fall back to Germany here: https://github.com/guardian/support-frontend/blob/a78a62b1839512e1cbde8aa952e0032624e5bf6e/support-frontend/assets/helpers/internationalisation/country.js#L560

Reply:
I've changed this now because it seems to work fine when in CODE or PROD, because those cookies are set. It does make it fiddly to test locally though, you need to use a VPN and/or manually set/update the cookie values.

Reply:
As discussed in the chat, the handling of country is overcomplicated currently.
If you're not on vpn then fastly's country cookie will not match with the CountryGroup, hence the default of germany.
For testing we can use the `country` querystring param